client/app/scripts/utils/web-api-utils.js

```diff
@@ -171,13 +177,13 @@ export function getNodeDetails(topologyUrlsById, currentTopologyId, options, nod
   const obj = nodeMap.last();
   if (obj && topologyUrlsById.has(obj.topologyId)) {
     const topologyUrl = topologyUrlsById.get(obj.topologyId);
-    let urlComponents = [topologyUrl, '/', encodeURIComponent(obj.id)];
+    let urlComponents = [API_PATH, '/', trimStart(topologyUrl, '/api'), '/', encodeURIComponent(obj.id)];
```

Comment:
We could, but we still have to trim off the leading slash from the topologyUrl like we do in `getAllNodes()`. `urlComponents` used to have a `.substr(1)` on it to remove the leading slash.